shell/dialog/ScalePoolDownDialog.vue

```diff
@@ -0,0 +1,118 @@
+<script>
+import { Card } from '@components/Card';
+import { alternateLabel } from '@shell/utils/platform';
+import { Checkbox } from '@components/Form/Checkbox';
+
+export default {
+  components: { Card, Checkbox },
+
+  props: {
+    resources: {
+      type:     Array,
+      required: true
+    }
+  },
+
+  data() {
+    const allToDelete = Array.isArray(this.resources) ? this.resources : [this.resources];
+
+    return {
+      allToDelete,
+      type:               this.$store.getters['type-map/labelFor'](allToDelete[0].schema, allToDelete.length),
+      promptConfirmation: true
+    };
+  },
+  computed: {
+    machinenName() {
+      return this.resources[0].id.split('/')[1];
+    },
+    protip() {
+      return this.t('promptRemove.protip', { alternateLabel });
+    },
+  },
+  mounted() {
+    const showScalePoolPrompt = this.$cookies.get('scalePoolPrompt');
+
+    if ( showScalePoolPrompt === undefined ) {
+      this.promptConfirmation = true;
+      this.$cookies.set('scalePoolPrompt', true);
+    } else {
+      this.promptConfirmation = showScalePoolPrompt;
+    }
+  },
+
+  methods: {
+    close() {
+      this.$emit('close');
+    },
+
+    update(e) {
+      if (this.promptConfirmation) {
```

Comment:
The value `promptConfirmation` is already a boolean, so you dont need a condition logic. If the value may be nullish, then you can enforce it with `Boolean()` or double bangs (`!!`). 
Also you are not using the event, so you can avoid passing it.

E.g.:

```ts
update() {
  this.$cookies.set('scalePoolPrompt', !!this.promptConfirmation);
```